### PR TITLE
feat: navbar & club info tag component

### PR DESCRIPTION
**Pull request by in-mai-space**
# Description

[Navbar](https://trello.com/c/kIm2kRQm/16-mobile-navbar-component-fe)
[Club info tag](https://trello.com/c/z7LTqq3C/26-mobile-club-info-tag-component-fe)

Navbar and club info tag component.

# How Has This Been Tested?

No tests.

# Checklist

- [x] I have performed a self-review of my code
- [x] I have reached out to another developer to review my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: frontend/mobile/assets/svg/calendar.svg
Comment (alderwhiteford) [inline frontend/mobile/assets/svg/calendar.svg:1]: we should be able to pull these from an icons library. I believe that we decided on fontawesome

File: frontend/mobile/components/club-recruitment-info.tsx
Comment (alderwhiteford) [inline frontend/mobile/components/club-recruitment-info.tsx:1]: @TylerSchaefer4 do we want to host our components beneath design-system?

File: frontend/mobile/assets/svg/calendar.svg
Comment (garrettladley) [reply frontend/mobile/assets/svg/calendar.svg:1]: +1

File: frontend/mobile/app/(app)/(tabs)/_layout.tsx
Comment (garrettladley) [inline frontend/mobile/app/(app)/(tabs)/_layout.tsx:1]: @in-mai-space could we resolve the errors and warnings in this file?

File: frontend/mobile/components/club-recruitment-info.tsx
Comment (garrettladley) [inline frontend/mobile/components/club-recruitment-info.tsx:1]: same idea here, could we resolve lint warnings? i believe we can ignore inline styles warnings

**Review by garrettladley — CHANGES_REQUESTED** [review]
some nits and looking for @TylerSchaefer4's feedback. keep up the great work @in-mai-space 👑

File: frontend/mobile/components/club-recruitment-info.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    text?: string | undefined,
+}
+
+const capitalize = (str: string) => {
+    if (!str) return '';
+    return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+export const RecruitmentInfo = ({recruitmentText, color, recruitmentCycle, recruitingType, isRecruiting = false}: RecruitmentInfoProps) => {
+    const RecruitmentItem = ({icon, title, text}: RecruitmentItemProps) => {
```
Comment (TylerSchaefer4) [inline frontend/mobile/components/club-recruitment-info.tsx:29]: Piggy backing off the lint warning, define `RecruitmentItem` as it's own component in it's own file and then import it to use here

File: frontend/mobile/app/(app)/(tabs)/_layout.tsx [hunk 2]
```diff
@@ -2,28 +2,90 @@ import React from 'react';
...
+
 const Layout = () => {
     return (
-        <Tabs screenOptions={{ tabBarShowLabel: false }}>
+        <Tabs
+            screenOptions={{
+                tabBarShowLabel: true,
+                tabBarStyle: {
+                    borderTopWidth: 1,
+                    borderTopColor: '#00000066',
```
Comment (alderwhiteford) [inline frontend/mobile/app/(app)/(tabs)/_layout.tsx:47]: avoid hardcoding colors. Either use black or add this to the color theme config

**Pull request merged**